File: CoreJava/man-boy-app/src/com/techlabs/empolyee/Manager.java
package com.techlabs.empolyee;

public class Manager extends Employee{
	
	private double hra;
	private double ta;
	private double sal;
	
	public Manager(int e, String n, double b) {
		super(e, n, b);
	}
	
	@Override
	public double calNetSalary() {
		hra=basic*0.5;
		ta=basic*0.4;
		sal=basic+hra+ta;
		return sal;
	}
	
}

File: C#/OOP/InheritanceExample/InheritanceExample/Manager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InheritanceExample
{
    class Manager : Employee
    {
        private double hra;
        private double ta;
        private double sal;

        public Manager(int e, String n, double b)
            : base(e, n, b)
        {
        }


        public override double calNetSalary()
        {
            hra = basic * 0.5;
            ta = basic * 0.4;
            sal = basic + hra + ta;
            return sal;
        }
    }
}
